storage/reports.go

```diff
@@ -23,4 +27,11 @@ func (db *store) GetRunningReportForUser(userID int64) (*entities.Report, error)
 	var report = new(entities.Report)
 	err := db.Where("user_id = ? and status = ?", userID, entities.StatusInProgress).Find(report).Error
 	return report, err
-}
\ No newline at end of file
+}
+
+// GetNumberOfReportsForDateTime returns number of reports for user id and datetime.
+func (db *store) GetNumberOfReportsForDateTime(userID int64, time time.Time) (int64, error) {
+	var count int64
+	err := db.Model(entities.Report{}).Where("user_id = ? and created_at = ?", userID, time.Format("2006-01-02-15:04:05")).Count(&count).Error
```

Comment:
The time format is weird here, we should fetch the number of reports for a given day right?